scripts/index_location

```diff
@@ -5,13 +5,16 @@ from sqlalchemy.orm import sessionmaker
 from wxfs.indexer import index_location
 
 
-def main(dsn, directories):
+def main(dsn, version, directories):
+    """Indexes all files in a single directory. The files must all correspond to
```

Comment:
> Indexes all files in a single directory.

Is that true? Doesn't this procedure index all the files in a collection of directories, where each individual directory must satisfy those conditions?

Reply:
You're remembering `index_location_collection`, the other indexing script. There are two indexing scripts; one indexes a single directory representing a location, and one indexes a "master" directory that contains many such location directories.